fission-core/library/Fission/Internal/UTF8.hs

```diff
@@ -5,116 +5,37 @@ module Fission.Internal.UTF8
   , putTextLn
   , putNewline
   , displayLazyBS
-  , toBase58Text
   , fromRawBytes
-  , stripOptionalPrefix
-  , stripOptionalPrefixBS
-  , stripOptionalPrefixLazyBS
-  , stripOptionalSuffix
-  , stripOptionalSuffixBS
-  , stripOptionalSuffixLazyBS
-  , stripPadding
-  , stripQuotes
-  , stripQuotesBS
-  , stripQuotesLazyBS
-  , stripN
-  , stripNBS
-  , stripNewline
+  , toBase58Text
   , textShow
   , wrapIn
+  , module Web.UCAN.Internal.UTF8
```

Comment:
I wasn't quite sure what to do with these `.Internal` stuff that we've been depending on in fission-core.

So what I did was split it up, you can see me removing stuff from the fission-core `.Internal.UTF8` module here, and instead reexporting the `Web.UCAN.Internal.UTF8` module, where that stuff was moved to.

Reply:
Hmm good point! A bunch of this stuff is actually now available in other libraries. We could switch to those, or break out a `fission-utils` or something 🤔